client/.eslintrc

```diff
@@ -18,6 +18,7 @@
    "react/react-in-jsx-scope": "off",
    "react/prop-types": "off",
    "react-hooks/rules-of-hooks": "error",
-   "react-hooks/exhaustive-deps": "error"
+   "react-hooks/exhaustive-deps": "error",
+   "prettier/prettier": ["off", { "singleQuote": true }]
```

Comment:
Is this turning off Prettier on the client side?

Reply:
It was my only way to fix the prettier problem before but it's no longer an issue, so I will remove it